src/screenHeights.ts

```diff
@@ -10,8 +8,10 @@ export const sfvcScreens = {
       "1.25": [753, 748, 744, 738],
       "1.5": [752, 749, 744, 738],
       "1.75": [753, 747, 744, 739],
-      "2": [752, 748, 744], // 738 removed as same in Safari
-      "2.5": [753, 748], // 745, 738 removed as same in Safari
+      "2": [752, 748, 744],
+      // 738 removed as same in Safari
```

Comment:
I don't think these comments are right anymore, we might either delete them or put them a line above